torch/_torch_docs.py

```diff
@@ -8810,6 +8810,33 @@ def merge_dicts(*dicts):
     tensor([ 4,  5,  8])
 """.format(**common_args))
 
+add_docstr(torch.take_along_dim,
+           r"""
+take_along_dim(input, indices, dim, *, out=None) -> Tensor
+
+Returns values from the :attr:`input` at the given 1-d indices along the given dim.
+
+.. note::
+    This function is similar to NumPy's `take_along_axis`.
+    See also :func:`torch.gather`.
+
+Args:
+    {input}
+    indices (LongTensor): the indices into tensor
+    dim (int): dim to take 1d slices along.
```

Comment:
"dimension to select along"

Let's not use terms like "slice" because they're not defined.